### DISCUSSION TITLE
BUILD_EPOCH Error

### DISCUSSION THREAD

**Discussion by Manu68350** [tianocore/edk2#5213] [General]
Hello, When i compile for RPI4. i have any error in VirtualRealTimeClockLib.c
in line 89 :  EpochSeconds = BUILD_EPOCH

I have make any chnage  for test to  EpochSeconds = 0
And this compile all....

BUILD_EPOCH ???? any way for work what this ???

**Comment by dan-bryant** [comment]
I can clear the `BUILD_EPOCH` error with VS2022 using the following, but I still get errors further along.

```
set CL=/DBUILD_EPOCH=1743197409
stuart_ci_build -c .pytool/CISettings.py -a X64 -t DEBUG TOOL_CHAIN_TAG=VS2022
```

But it simply progresses further to errors in `libfdt_env.h`
